### PR TITLE
Upgrade CLI Feature

### PR DESCRIPTION
**Pull request by davidkel**

### CODE REVIEW

File: packages/composer-cli/test/network/upgrade.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+const Admin = require('composer-admin');
+const InstallCmd = require('../../lib/cmds/network/upgradeCommand.js');
+const CmdUtil = require('../../lib/cmds/utils/cmdutils.js');
+
+require('chai').should();
+
+const chai = require('chai');
+const sinon = require('sinon');
+require('sinon-as-promised');
```
Comment (sstone1) [inline packages/composer-cli/test/network/upgrade.js]: The `sinon-as-promised` require needs to be removed due to the sinon related PR earlier.

File: packages/composer-runtime/lib/engine.js [hunk 2]
```diff
@@ -90,6 +90,19 @@ class Engine {
         const method = 'init';
         LOG.entry(method);
         LOG.entry(method, context, fcn, args);
+
+        // chaincode was upgraded, no change to business network and obviously
+        // nothing the runtime can do to stop it.
+        if (fcn === 'upgrade') {
+            LOG.info('runtime has been upgraded');
```
Comment (sstone1) [inline packages/composer-runtime/lib/engine.js]: First argument to the logging calls should be `method` - whilst you're here, can you remove the duplicate `LOG.entry` at the top please?

Comment (davidkel) [inline packages/composer-runtime/lib/engine.js]: @sstone1 Thanks Simon, missed that one. Figured I would need to do the sinon as well after seeing yours :-)

**Comment by sstone1** [comment]
Fixes look good to me 👍

**Pull request merged**